javascript/javascript1/week4/infoCreditCard.js

```diff
@@ -0,0 +1,26 @@
+//Credit card info
+
+let number = '4781321334789876';
+
+function GetCardType(number)
```

Comment:
you should be careful lifting code from Stackoverflow, it's not always correct.  In this case it's pretty close, but most of the same behavior could have been done with startswith, regular expressions are hard :confounded: 

```
number.startswith('4')
```
or 
```
number[0] === "4"
```